src/interactive/results.ts

```diff
@@ -261,17 +261,27 @@ export function removeResult(target: Result) {
     return results.splice(results.indexOf(target), 1)
 }
 
+export function removeErrorResult(target: Result) {
+    target.remove()
+    const targetIndex = getErrorResults().indexOf(target)
+    return stackFrameHighlights.highlights.splice(targetIndex, 1)
+}
```

Comment:
FWIW, I quite like the Juno behaviour where the stacktrace is attached to the last result. So when you clear the result that caused the error display the error trace is also cleared.

Reply:
yeah, I'm just trying to implement that, but I'm still not sure what the best way to implement that (maybe we need to keep the last result as a global object and subscribe to its validation or something like that ?) I've not reviewed how you implemented that in Juno, but I'm very welcome if you want to implement the logic.
If so I will tackle on implementing utility commands that can be invoked as a hover links :)

Reply:
Done.

Reply:
your implementation looks sooo valid... thanks !